ion/bitstream.go

```diff
@@ -660,6 +660,11 @@ func (b *bitstream) ReadBytes() ([]byte, error) {
 		panic("not a lob")
 	}
 
+	// A0 and 90 are special cases, denoting an empty blob and an empty clob respectively, with b.len == 0.
+	if b.len == 0 {
+		return []byte{}, nil
```

Comment:
This misses the update to `b.state` down below; that seems like it might be a bug?

Reply:
Good point, I'll address it.
Thanks!

Reply:
PR #87